### DISCUSSION TITLE
Youtube Downloads

### DISCUSSION THREAD

**Discussion by Altonhendricks** [subhra74/xdm#1083] [General]
Please let me know if YouTube stoped downloads using XDM.

**Comment by Ju-5099** [comment]
hello, I use XDM very often and I would like to know if the latest version present on the microsoft store for windows could regain functions present on older versions that were really very useful like the ability to convert video files ( .ts to .mp4 )before downloading and also integrate the old download fold-down bar which was one of the best assets of the software.
XDM has lost some of its potential and capabilities and I find the software weaker, less efficient and less fast now ( 8.0.26 BETA ) compared to older versions (7.2.11 and previous versions). Otherwise, it would be genial to be able to download from youtube as in the previous versions too.

XDM is awesome and deserves to become the best software on the market, and for that it needs all its best assets available

**Reply by smestarz** [reply]
Seems youtube changed something. XDM does not download youtube videos, 
There is other software that helps me download, but I am very used to XDM as it does it very fast and reliably
